server/yarn.lock

```diff
@@ -1019,20 +1019,26 @@ color-name@1.1.3:
   resolved "https://registry.yarnpkg.com/color-name/-/color-name-1.1.3.tgz#a7d0558bd89c42f795dd42328f740831ca53bc25"
   integrity sha1-p9BVi9icQveV3UIyj3QIMcpTvCU=
 
+<<<<<<< HEAD
```

Comment:
All of this doesn't seem like it should be there.

Reply:
I did a git pull origin master and I'm not getting any conflicts.

Reply:
Nevermind I found them